content/SCALE/GettingStarted/Getting Support/FileIssueSCALE.md

```diff
@@ -41,3 +41,23 @@ Attach any applicable screenshots and click **Save**.
 After the ticket generates, the **Ticket** confirmation dialog opens. Click **Open Ticket** to view the ticket in Jira.
 
 ![GeneralSettingsSCALEJiraTicketLink](/images/SCALE/SystemSettings/GeneralSettingsSCALEJiraTicketLink.png "SCALE General Settings Jira Ticket Link")
+
+### Creating an Issue In Jira
+
+If logging into Jira and creating an issue ticket, download a system debug after the issue occurs and before you reboot the system to capture log files necessary to troubleshoot and diagnose system issues.
+
+Always store debug files in a secure location.
+
+You can attach the debug directly to the issue ticket if you are not concerned about others accessing the debug file. 
+Issues tickets with debug files are marked private during the triage process but this can lead to duplicated reports of the same issue when another user cannot find an existing report that is marked private. 
+Duplicate reports of the same issue are closed to allow Engineering staff to focus their efforts on the original issue report.
+
+To protect user data privacy, the TrueNAS Jira project sends the issue reporter a link to a private file attachment area where they can upload their debug file and link the upload ticket to their issue report ticket.
```

Comment:
I'd suggest swapping this paragraph with the one currently at line 51 then frame the opening of that one more as something like _We do not recommend that users attach debugs directly..._ 

Keeps the focus on the positive/preferred action

Reply:
change made in the next commit.